### PR TITLE
fixed issue766

### PR DESCRIPTION
**Pull request by brianzhangrong**
[fixed] issue766:error code definition

### CODE REVIEW

**Review by brianzhangrong — COMMENTED** [review]
remove target

**Comment by wushengyeyouya** [comment]
There are a lot of history commits attached to this commit, please resubmit it

File: README.md [hunk 1]
```diff
@@ -76,24 +76,10 @@ Linkis services could be divided into three categories: computation governance s
 Below is the Linkis architecture diagram. You can find more detailed architecture docs in [Linkis-Doc/Architecture](https://github.com/WeBankFinTech/Linkis-Doc/tree/master/zh_CN/Architecture_Documents).
 ![architecture](images/Linkis_1.0_architecture.png)
 
-Based on Linkis the computation middleware, we've built a lot of applications and tools on top of it in the big data platform suite [WeDataSphere](https://github.com/WeBankFinTech/WeDataSphere). Below are the currently available open-source projects.
```
Comment (peacewong) [inline README.md:79]: Hello, this PR introduces old commits

**Comment by wushengyeyouya** [comment]
New PR has submitted, close this one.

**Pull request closed without merging**